compiler/z/codegen/OMRCodeGenerator.hpp

```diff
@@ -256,6 +256,7 @@ class OMR_EXTENSIBLE CodeGenerator : public OMR::CodeGenerator
 
 protected:
 
+   List<TR_Pair<TR::Node, int32_t> > _ialoadUnneeded; 
```

Comment:
This should be a private field as it is not currently used by any subclasses of `CodeGenerator`. Could we move it down under the `private:` section where all the other fields are?